src/realm/index_string.cpp

```diff
@@ -2053,15 +2045,13 @@ void StringIndex::verify_entries(const ClusterColumn& column) const
 
 namespace {
 
-namespace {
-
 bool is_chars(uint64_t val)
 {
     if (val == 0)
         return true;
     if (is_chars(val >> 8)) {
         char c = val & 0xFF;
-        if (!c || std::isalpha(c)) {
+        if (!c || std::isprint(c)) {
```

Comment:
NIT: `bool is_chars`  seems a little bit misleading, probably is_printable() is better.